### ISSUE TITLE
Missing `charset=utf-8` in mimetype in registry.txt

### ISSUE THREAD

**Issue by alesrosina** [Tudmotu/gnome-shell-extension-clipboard-indicator#417]
After upgrading to Gnome 45, all new texts are added to `registry.txt` as follows:

```json
  {
    "favorite": false,
    "mimetype": "text/plain",
    "contents": "some text that should be pastable"
  },
  {
    "favorite": false,
    "mimetype": "text/plain",
    "contents": "some more examples"
  }
```
Note missing `"mimetype": "text/plain;charset=utf-8",`. With this setting, when sthing in text is selected, you cannot paste anything (when doing paste, it seems like it's empty). 

If I select older record or add `charset=utf-8` manually to `registry.txt`, save it and then reselect (click on item in extension), it will be able to paste.

I should add, that this mostly happens when copying from Visual Studio Code and Firefox.

---

System Info:
Ubuntu 23.10, Gnome 45, vscode 1.84.0, Firefox 119.0

**Comment by Tudmotu** [comment]
Thank you for reporting this.
Yes, since GNOME 45 mimetypes have been added and it has been causing issues.

I am trying to figure out a way to fix this.

**Comment by kiamlaluno** [comment]
I edited the registry.txt file and changed `"mimetype": "text/plain"` to `"mimetype": "text/plain;charset=utf-8"`. After I switched off, and back on, Clipboard Indicator started to copy again the text.

**Comment by megaconfidence** [comment]
Thanks for sharing this @kiamlaluno, I was almost going crazy

Any progress on a permanent fix @Tudmotu?

**Comment by Tudmotu** [comment]
Yeah, but slow progress since I've been preoccupied with other stuff.

Hopefully I can find time to work on it next week and release a new version.

**Comment by megaconfidence** [comment]
Awesome bro, thank you. I always appreciate your quick fixes on issues like these!

**Comment by Tudmotu** [comment]
Short update, seems like this is an issue with GNOME. More details in this comment:
https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/issues/425#issuecomment-1892821774

**Comment by kiamlaluno** [comment]
I wish my previous comment said where the registry.txt file is. I am trying to find it, but I do not remember where it is.

**Comment by Tudmotu** [comment]
@kiamlaluno it should be in `~/.cache/clipboard-indicator@tudmotu.com`

**Comment by kiamlaluno** [comment]
@Tudmotu Thank you.
